src/main/java/com/crowdin/client/glossaries/model/SearchConcordanceRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.crowdin.client.glossaries.model;
+
+import lombok.Data;
+
+@Data
+public class SearchConcordanceRequest {
+    private String sourceLanguageId;
+    private String targetLanguageId;
+    private String expression;
+    private Integer offset;
+    private Integer limit;
```

Comment:
Please remove the `offset` and `limit` here (upcoming API update, it was added by mistake)

Reply:
How about the response itself? Should it be included there?

Reply:
let me check

Reply:
@andrii-bodnar so, no more changes from my side I guess